handlers/bounties.go

```diff
@@ -137,9 +137,9 @@ func MigrateBounties(w http.ResponseWriter, r *http.Request) {
 				migrateBountyFinal.Award = Award
 			}
 
-			Price, ok5 := migrateBounty["price"].(string)
+			Price, ok5 := migrateBounty["price"].(uint)
 			if !ok5 {
-				migrateBountyFinal.Price = "0"
+				migrateBountyFinal.Price = 0
```

Comment:
so I think this only happened when we did that initial migration so I don't think this will modify any existing bounties so they will continue to be strings. Not sure if we should do this change, we should honestly remove this migration code aswell since if we run it again we will have duplicate bounties which is not good